### PR TITLE
custom links: collect tool links and assemble into html in gather-extra

### PR DESCRIPTION
**Pull request by vrutkovs**
This would let us create one file for all tools used during the run. Currently Loki and PromeCIeus are integrated.

[Example job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/18260/rehearse-18260-pull-ci-operator-framework-operator-marketplace-release-4.9-e2e-aws-upgrade/1389692608615288832).

Caveats:
* [ ] `html` lens is an iframe, so use "Open in New Tab/Window" on the links, otherwise it would reuse the iframe to display (press Back if it happened)
* [ ] Chrome (and Safari apparently) prevent the script from reading refererer, so PromeCIeus link is incomplete.

### CODE REVIEW

File: ci-operator/step-registry/gather/extra/gather-extra-commands.sh [hunk 1]
```diff
@@ -131,19 +131,8 @@ echo "Snapshotting prometheus (may take 15s) ..."
...
-{
-    item.href="https://promecieus.dptools.openshift.org/?search="+window.location.href;
-    return true;
-}
-</script>
-<a onclick="return buildURL(this)" href="">PromeCIeus</a>
-</body>
-</html>
+cat >> ${ARTIFACT_DIR}/custom-links.txt << EOF
+<a target="_blank" href=""https://promecieus.dptools.openshift.org/?search=https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/${JOB_NAME_SAFE}/${BUILD_ID}"">PromeCIeus</a>
```
Comment (wking) [inline ci-operator/step-registry/gather/extra/gather-extra-commands.sh:135]: `""` -> `"` for the `href`?

File: ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh [hunk 2]
```diff
@@ -515,11 +515,7 @@ echo "Promtail manifests created, the cluster can be found at https://grafana-lo
...
 if [[ -f "/usr/bin/python3" ]]; then
   ENCODED_INVOKER="$(python3 -c "import urllib.parse; print(urllib.parse.quote('${OPENSHIFT_INSTALL_INVOKER}'))")"
-  cat >> ${ARTIFACT_DIR}/custom-link-loki.html << EOF
-  <html>
-  <body>
-  <a target="_blank" href="https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%5B%22now-24h%22,%22now%22,%22Grafana%20Cloud%22,%7B%22expr%22:%22%7Binvoker%3D%5C%22${ENCODED_INVOKER}%5C%22%7D%22%7D%5D">Loki</a>
-  </body>
-  </html>
+  cat >> ${ARTIFACT_DIR}/custom-links.txt << EOF
+  <a target="_blank" href="https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%5B%22now-24h%22,%22now%22,%22Grafana%20Cloud%22,%7B%22expr%22:%22%7Binvoker%3D%5C%22${ENCODED_INVOKER}%5C%22%7D%22%7D%5D">Loki<i class="material-icons" style="font-size: 1em; vertical-align: middle; padding-left: 3px;">open_in_new</i></a>
```
Comment (wking) [inline ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh:519]: Does this need an `| unpack` like the `echo` line from above?

Comment (vrutkovs) [reply ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh:519]: Good idea

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**